### PR TITLE
Epstein Homework 1 Submission 

### PR DESCRIPTION
**Pull request by warrenepstein**
---
name: Warren Casey Epstein
the file that is the complete version is "44c8a3e"
about: Used for students submitting a pull request for a homework assignment, 

I actually had a very hard time committing. I was able to figure it out by restoring (git reset --hard) and then rewriting the last bit of code that I was having a hard time getting to work. I was helped in figuring this out by stack exchange. Hopefully, for the next assignment I will have an easier time committing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01
+# About Warren Casey Epstein
+## Undergraduate Student at the University of Chicago
+
+
+## Rising Third-Year in the College
```
Comment (pthelen124) [inline README.md:6]: In the future, you should try to leave a blank line after headers. It's not an issue here, but once you begin working with code chunks in RMarkdown files later, this practice has the potential to save you some formatting troubles.

File: README.md [hunk 2]
```diff
@@ -1 +1,23 @@
...
+I am a rising third-year in the College, double majoring in **political science** and **economics** *with a specialization in business economics*. I will also hopefully minor in *near eastern languages and civilizations*.
+
+
+## Backround
+I am orginally from Fairfield, Connecitcut, but went to high school in Riverdale New York at SAR High School.
+
+
+## General Hobbies, interests, and likes/dislikes
+1. Eating all things pumpkin - fall is my favorite time of the year. **pumpkin ice cream**, **pumpkin spice lattes**, **pumpkin pie** the options are endless
```
Comment (pthelen124) [inline README.md:15]: You don't actually have to manually number lists. GitHub will automatically renumber the items: 

```r
1. Eating all things pumpkin
1. Sailing
1. Hiking
```

1. Eating all things pumpkin
1. Sailing
1. Hiking

File: README.md [hunk 3]
```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01
+# About Warren Casey Epstein
```
Comment (pthelen124) [inline README.md:2]: Great use of a title and subheaders using a variety of `#` to break up the flow of your document.

File: README.md [hunk 4]
```diff
@@ -1 +1,23 @@
...
+## General Hobbies, interests, and likes/dislikes
+1. Eating all things pumpkin - fall is my favorite time of the year. **pumpkin ice cream**, **pumpkin spice lattes**, **pumpkin pie** the options are endless
+2. Sailing - when the never ending Chicago winter is over
+3. Hiking - also when the aforementioned Chicago winter is over
+
+## Some more favorites that are now affected by the Chicago Winter or season - pumpkin seems to go out sytle in the winter
+
+1. *To Kill a Mockingbird* by Harper Lee is my favorite book
+2. I enjoy reading *The Economist* and may have paid more money than I should have for a print copy of the magazine at Midway airport last week
+3. My favorite musical is *Come from Away* ![Image of Come from Away] https://images.app.goo.gl/EqriepACZkfi2mZs8
```
Comment (pthelen124) [inline README.md:23]: Make sure that if you're linking an online photo, the page you've linked to is _intended_ to display that image. You linked to the Google Image Result here, so the photo itself won't render automatically.

Comment (djanvier) [inline README.md:23]: Unfortunately, the caption does not come out right. I think you needed to put the link to the image in parentheses for R to know it was an embedded image and not a link

File: README.md [hunk 5]
```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# hw01
+# About Warren Casey Epstein
+## Undergraduate Student at the University of Chicago
+
+
+## Rising Third-Year in the College
+I am a rising third-year in the College, double majoring in **political science** and **economics** *with a specialization in business economics*. I will also hopefully minor in *near eastern languages and civilizations*.
```
Comment (djanvier) [inline README.md:7]: I like how you use emphasis to make a distinction between majors and minors. Very creative!

**Review by pthelen124 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |   x           |            |
| **Coding strategy**         |           |    x          |            |
| **Presentation**            |           |   x           |            |
| **Achievement, creativity** |           |    x          |            |
| **Ease of access**          |   x        |              |            |

* In the future, it’s actually better if you preview your README file on GitHub. Doing a local preview creates an unnecessary `html` file, and using the `stage/commit/pull/push` method is good practice anyway. 
* Don’t be discouraged by issues with the `stage/commit/pull/push` method. It gets easier with practice. As you said in your reflection, resources like **Stack Exchange** can be really helpful.

* Overall, nice job on this assignment!

**Review by djanvier — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |    x          |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |           |        x      |            |
| **Achievement, creativity** |           |       x       |            |
| **Ease of access**          |       x    |              |            |

Remarks:

Overall, nice work! I would experiment even more with the syntax and see what other creative ways you could use to convey the same information.

File: README.md (hunk 4 above)
Comment (pthelen124) [reply README.md:23]: I also just noticed: in order to link to a page, the URL needs to be inside parentheses following the bracketed text (no space): 

 ```r
![Image of Come from Away](https://images.app.goo.gl/EqriepACZkfi2mZs8)
```

Renders as: 
Image of Come from Away

**Comment by wintong** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |           |    X          |            |
| **Presentation**            |           |     X         |            |
| **Achievement, creativity** |  X         |              |            |
| **Ease of access**          |       X    |              |            |

Interesting information provided (I also like pumpkin-y things!) - I think it was a bit harder to figure out the code because of the multiple commits (or maybe I don't know where these are yet) but otherwise nice content!

**Pull request closed without merging**